bin/inx-chronicle/src/metrics.rs

```diff
@@ -56,25 +69,31 @@ impl Default for MetricsConfig {
         }
     }
 }
-struct UpdateDbMetrics {
+struct GapsMetric {
     db: MongoDb,
-    size: Gauge<u64>,
 }
 
 #[async_trait]
-impl Task for UpdateDbMetrics {
+impl Task for GapsMetric {
```

Comment:
Let's remove that until we have a better way to query gaps.

Reply:
Seems like this coming with #495 .